### PR TITLE
doc: Updating VLAN schema

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: doc/swss-schema.md [hunk 1]
```diff
@@ -101,10 +101,13 @@ For example (reorder output)
...
-    vland_id            = DIGIT 0-4095
-    admin_status        = BIT ; administrative status for vlan up or down
-    attach_to           = PORT_TABLE.key
+    key                 = VLAN_TABLE:vlanid	; DIGIT 0-4095
+    admin_status        = "down" / "up"		; admin status
+    oper_status		= "down" / "up"		; operating status
+    
+    key			= VLAN_TABLE:vlanid:membername	; vlan member
+    oper_status		= "down" / "up"		; operating status
+
```
Comment (lguohan) [inline doc/swss-schema.md]: convert tab to spaces.

File: doc/swss-schema.md [hunk 2]
```diff
@@ -101,10 +101,13 @@ For example (reorder output)
...
-    key                 = DEVICE_NAME    ; must be unique across PORT,INTF,VLAN,LAG TABLES
-    vland_id            = DIGIT 0-4095
-    admin_status        = BIT ; administrative status for vlan up or down
-    attach_to           = PORT_TABLE.key
+    key                 = VLAN_TABLE:vlanid	; DIGIT 0-4095
+    admin_status        = "down" / "up"		; admin status
+    oper_status		= "down" / "up"		; operating status
+    
+    key			= VLAN_TABLE:vlanid:membername	; vlan member
+    oper_status		= "down" / "up"		; operating status
```
Comment (lguohan) [inline doc/swss-schema.md]: why oper status is needed here? what's the difference between this one and the physical port oper status?

**Comment by lguohan** [comment]
:clock1:

**Comment by stcheng** [comment]
Updated and remove the oper_status.

**Comment by lguohan** [comment]
What about the oper_status for the vlan_member?

---
In reply to: [223851040](https://github.com/Azure/sonic-swss/pull/33#issuecomment-223851040) [](ancestors = 223851040)

File: doc/swss-schema.md [hunk 3]
```diff
@@ -101,10 +101,12 @@ For example (reorder output)
 ###VLAN_TABLE
     ;Defines VLANs and the interfaces which are members of the vlan
     ;Status: work in progress
-    key                 = DEVICE_NAME    ; must be unique across PORT,INTF,VLAN,LAG TABLES
-    vland_id            = DIGIT 0-4095
-    admin_status        = BIT ; administrative status for vlan up or down
-    attach_to           = PORT_TABLE.key
+    key                 = VLAN_TABLE:vlanid	; DIGIT 0-4095
+    admin_status        = "down" / "up"		; admin status
```
Comment (lguohan) [inline doc/swss-schema.md]: I still see tabs, can you convert them into spaces?

Comment (stcheng) [inline doc/swss-schema.md]: updated

**Pull request merged**